tests/publiccloud/patch_and_reboot.pm

```diff
@@ -25,18 +26,21 @@ sub run {
 
     my $cmd_time = time();
     my $ref_timeout = check_var('PUBLIC_CLOUD_PROVIDER', 'AZURE') ? 3600 : 240;
-    kill_packagekit($args->{my_instance});
+    # pkcon not present on SLE-micro
+    kill_packagekit($args->{my_instance}) unless (is_sle_micro);
     $args->{my_instance}->ssh_script_retry("sudo zypper -n --gpg-auto-import-keys ref", timeout => $ref_timeout, retry => 6, delay => 60);
     record_info('zypper ref time', 'The command zypper -n ref took ' . (time() - $cmd_time) . ' seconds.');
     record_soft_failure('bsc#1195382 - Considerable decrease of zypper performance and increase of registration times') if ((time() - $cmd_time) > 240);
-
-    ssh_fully_patch_system($remote);
-
+    if (is_sle_micro) {
+        ssh_fully_patch_system_micro($args->{my_instance});
+    } else {
+        ssh_fully_patch_system($remote);
+    }
     record_info('UNAME', $args->{my_instance}->ssh_script_output(cmd => 'uname -a'));
     $args->{my_instance}->ssh_assert_script_run(cmd => 'rpm -qa > /tmp/rpm-qa.txt');
     $args->{my_instance}->upload_log('/tmp/rpm-qa.txt');
 
-    $args->{my_instance}->softreboot(timeout => get_var('PUBLIC_CLOUD_REBOOT_TIMEOUT', 600));
+    $args->{my_instance}->softreboot(timeout => get_var('PUBLIC_CLOUD_REBOOT_TIMEOUT', 600)) unless (is_sle_micro);
```

Comment:
Can you somehow comment this?

Reply:
`is-sle-micro` cases have **reboot** already done in the `fully_...` routine, otherwise still needed todo. 
But please check the last version pushed.